app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceAdapter.kt

```diff
@@ -280,23 +327,30 @@ class DashboardAdapter(
     }
 }
 
-sealed class DashboardModel {
-    object Header : DashboardModel()
+sealed class InsuranceModel {
+    object Header : InsuranceModel()
+
+    sealed class InfoBox : InsuranceModel() {
```

Comment:
This one no longer needs to be a sealed class, as there is only one type represented by it - the `Renewal`